src/services/connectors/verida.ts

```diff
@@ -125,6 +128,16 @@ export class VeridaService {
 			credentialSchema,
 			credentialData: credential,
 		};
-		await this.sendData(recipientDid, messageSubject, credentialRecord);
+		await this.sendData(environment, recipientDid, messageSubject, credentialRecord);
+	}
+
+	static splitVeridaDid(did: string) {
```

Comment:
Let's add Verida did validation at the controller validation level.

Reply:
@lampkin-diet there was a VeridaDIDValidator already used that and removed this. Thanks